scss/layout/_header.scss

```
@@ -0,0 +1,17 @@
+.header {
+  width: 100%;
+  height: 80px;
+  z-index: 4;
+  font-weight: 600;
+
+  &-wrapper {
+    height: 80px;
+    @include flex($justify: space-between, $align: center)
```

```suggestion
    @include flex($justify: space-between, $align: center)
    height: 80px;
```
`@include` cứ đưa hết lên trên đầu nhé